chart/k8gb/values.yaml

```diff
@@ -29,12 +29,16 @@ k8gb:
     level: info # log level (panic,fatal,error,warn,info,debug,trace)
   splitBrainCheck: false
   metricsAddress: "0.0.0.0:8080"
-  runAsUser: 1000
+  securityContext:
```

Comment:
Not sure why are we making clarification comment below only for external-dns?

Reply:
fixed, it was by mistake